### PR TITLE
Fixes package validation errors for side-channels-related classes.

### PR DESCRIPTION
**Pull request by mmattar**
### Proposed change(s)

Fixes package validation errors for side-channels-related classes.

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [X] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: com.unity.ml-agents/Runtime/SideChannel/SideChannel.cs [hunk 1]
```diff
@@ -3,6 +3,11 @@
 
 namespace MLAgents
 {
+    /// <summary>
+    /// Side channels provides an alternative mechanism of sending/receiving data from Unity
```
Comment (surfnerd) [inline com.unity.ml-agents/Runtime/SideChannel/SideChannel.cs]:
```suggestion
    /// Side channels provide an alternative mechanism of sending/receiving data from Unity
```

File: com.unity.ml-agents/Runtime/SideChannel/SideChannel.cs [hunk 2]
```diff
@@ -3,6 +3,11 @@
...
+    /// to Python that is outside of the traditional machine learning loop. ML-Agents provides
```
Comment (vincentpierre) [inline com.unity.ml-agents/Runtime/SideChannel/SideChannel.cs]:
```suggestion
    /// to Python that is outside of the traditional reinforcement learning loop. ML-Agents provides
```

Comment (mmattar) [reply com.unity.ml-agents/Runtime/SideChannel/SideChannel.cs]: Would prefer to keep it more generic.

File: com.unity.ml-agents/Runtime/SideChannel/SideChannel.cs (hunk 1 above)
Comment (mmattar) [reply com.unity.ml-agents/Runtime/SideChannel/SideChannel.cs]: 👍

**Pull request merged**